backend/src/main/java/de/echochamber/backend/config/UserServiceConfigProperties.java

```diff
@@ -0,0 +1,19 @@
+package de.echochamber.backend.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "echochamber.user-service")
```

Comment:
AvatarUrl ist immer docker.com , was ist da der Plan?

Reply:
Hmm? Die avatarUrl ist doch die url zum Profilbild oder verstehe ich da etwas falsch?

Reply:
du holst dir das über die Config dachte ich, daher prefix = "echochamber.user-service" . Wenn du in deine .yaml schaust ist das www.docker.de , aber vlt übersehe ich was. Es war ein langer PR, bin vlt etwas durcheinander gekommen